src/ClientData/include/ClientData/ScopeCollection.h

```diff
@@ -0,0 +1,45 @@
+
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_SCOPE_COLLECTION_H_
+#define CLIENT_DATA_SCOPE_COLLECTION_H_
+
+#include <cmath>
+
+#include "ClientData/ScopeIdProvider.h"
+#include "ClientData/ScopeStats.h"
+#include "ClientData/TimerTrackDataIdManager.h"
+
+namespace orbit_client_data {
+
+// ScopeCollection holds a subset of all Scopes in a capture keeping track of their stats and
+// ordered durations.
+class ScopeCollection {
```

Comment:
>  I think the orbit_client_data:: was unnecessary

Oh, that's great! 

> google style says not to use "using" in header files at all

Correct me if I am wrong, but it's less string in that regard. It says "Don't put an alias in your public API", moreover it explicitly says "local convenience aliases are fine in function definitions, private sections of classes, explicitly marked internal namespaces, and in .cc files". The point is to avoid namespace pollution for whoever includes the header, not to forbid aliases in headers altogether.

Reply:
Right not as strict as "never use in header files", but I think you generally need a reason to do it more than just "readability".

`Don't put an alias in your public API just to save typing in the implementation; do so only if you intend it to be used by your clients.` https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide-ext.md#Aliases